### DISCUSSION TITLE
New Color Widget

### DISCUSSION THREAD

**Discussion by Benno00** [FrSkyRC/ETHOS-Feedback-Community#4651] [Show and tell]
Hello Frsky team,
I have created a script for displaying values ​​that can change the color when the value is exceeded or undershot. And it can display the values ​​as a PNG scalable PNG graphic.
Here are some screens.
Perhaps there is interest in integrating these directly into ETHOS?
![image](https://github.com/user-attachments/assets/5af35b3c-651a-46de-a57c-906ae2277f15)
![image](https://github.com/user-attachments/assets/e6a6100f-006e-4766-8686-e55ed7e0fa6a)
![image](https://github.com/user-attachments/assets/8847d079-7622-4005-98a9-a5cbe975e7ed)
![image](https://github.com/user-attachments/assets/8ff9367c-1e7e-45ec-a668-3d7aa34f39bd)
![image](https://github.com/user-attachments/assets/3d5ceca4-5c4b-4103-8412-993a183cf2b1)
![image](https://github.com/user-attachments/assets/1dd6ff73-0212-4997-adf9-78cfbc973d4a)
![image](https://github.com/user-attachments/assets/409bd396-0a36-41ff-8551-8c07745b3741)

Here the dowload File as luac

**Comment by xet303** [comment]
Yes it would be great if it could be integrated into Ethos!
Great flexibility, compatibility with all sizes of widgets for a greatly improved aesthetics of the graphic interface.
Here are some other examples.
![screenshot-2025-01-04-26606](https://github.com/user-attachments/assets/b72b30db-5717-4ad0-a842-48b59fe1bc14)
![screenshot-2025-01-04-26650](https://github.com/user-attachments/assets/2f1178c3-fe40-449e-b9a9-0b0d06854c8e)
![screenshot-2025-01-04-26654](https://github.com/user-attachments/assets/c7a2e61a-1e00-422c-9f67-673034161d11)
![screenshot-2025-01-04-27457](https://github.com/user-attachments/assets/7b8f6f04-5e5c-45e0-88e3-dc1e80aecefd)

**Comment by Benno00** [comment]
Hello everyone,

here is the new version 2.6 with additional LiPo display, from ETHOS 2.6.1
Not for Kavan transmitters

![image](https://github.com/user-attachments/assets/be2868be-390c-4b4b-ba4c-95b213962955)

**Comment by Benno00** [comment]
Sorry the 2.6 version had a memory problem, here is the 2.7 version, memory problem has been fixed

**Comment by xet303** [comment]
Thank you,
Your version 2.7 seems to have no memory problems.
I tested it on Ethos 1.6.1.
![screenshot-2025-01-18-82964](https://github.com/user-attachments/assets/fdf4fad0-756e-4000-9d06-f10753de1e2c)

**Comment by xet303** [comment]
tested version 2.7.3 on Ethos 1.6.1. 
![screenshot-2025-01-20-70992](https://github.com/user-attachments/assets/d44d2ffe-6a80-4fc7-8558-2a19fdd61492)
No matter the lipo voltage we always get the first red bar on the battery.

**Comment by Benno00** [comment]
what are the settings in the Trigger Settings menu? 
A Lipo should have 4.2 V when it is full, yours only has 3.75 V so it is almost empty, charge the battery fully and then test.
Just tested again on my X20S.
![WhatsApp Bild 2025-01-21 um 19 28 15_15aef34d](https://github.com/user-attachments/assets/1012ba4e-3549-4aab-bf9e-d743284be477)

**Comment by xet303** [comment]
OK excuse me!
I had checked with other batteries approximately of the same voltage +- 0.15V.
So I had stayed with in mind that I use a Li-Ion pack forgetting that the Lipo widget is for LiPo packs! (min not the same)
![screenshot-2025-01-20-69242](https://github.com/user-attachments/assets/7f7d56ee-6968-44c2-8aed-7f1d611dcafb)
